dowhy/causal_estimator.py

```diff
@@ -126,6 +182,48 @@ def do(self, x):
     def construct_symbolic_estimator(self, estimand):
         raise NotImplementedError
 
+    def get_confidence_intervals(self, estimate):
+        '''
+            Find the confidence intervals corresponding to any estimator
+            This is done with the help of bootstrapped confidence intervals
+
+            For more details, refer to the following link:
+            https://ocw.mit.edu/courses/mathematics/18-05-introduction-to-probability-and-statistics-spring-2014/readings/MIT18_05S14_Reading24.pdf
```

Comment:
Class notes are good, but perhaps add a link to wikipedia or another authoritative source too?